rocketpy/plots/motor_plots.py

```diff
@@ -135,113 +135,41 @@ def exhaust_velocity(self, lower_limit=None, upper_limit=None):
-        """Plots I_22 of the motor as a function of time.
-
-        Parameters
-        ----------
-        lower_limit : float
-            Lower limit of the plot. Default is none, which means that the plot
-            limits will be automatically calculated.
-        upper_limit : float
-            Upper limit of the plot. Default is none, which means that the plot
-            limits will be automatically calculated.
-
-        Return
-        ------
-        None
-        """
-        self.motor.I_22.plot(lower=lower_limit, upper=upper_limit)
-
-    def I_33(self, lower_limit=None, upper_limit=None):
-        """Plots I_33 of the motor as a function of time.
-
-        Parameters
-        ----------
-        lower_limit : float
-            Lower limit of the plot. Default is none, which means that the plot
-            limits will be automatically calculated.
-        upper_limit : float
-            Upper limit of the plot. Default is none, which means that the plot
-            limits will be automatically calculated.
-
-        Return
-        ------
-        None
-        """
-        self.motor.I_33.plot(lower=lower_limit, upper=upper_limit)
-
-    def I_12(self, lower_limit=None, upper_limit=None):
-        """Plots I_12 of the motor as a function of time.
-
-        Parameters
-        ----------
-        lower_limit : float
-            Lower limit of the plot. Default is none, which means that the plot
+            Lower limit of the plot. Default is None, which means that the plot
             limits will be automatically calculated.
         upper_limit : float
-            Upper limit of the plot. Default is none, which means that the plot
+            Upper limit of the plot. Default is None, which means that the plot
             limits will be automatically calculated.
 
         Return
         ------
-        None
+        fig : matplotlib.figure.Figure
+            The matplotlib figure containing the inertia tensors plot.
         """
-        self.motor.I_12.plot(lower=lower_limit, upper=upper_limit)
-
-    def I_13(self, lower_limit=None, upper_limit=None):
-        """Plots I_13 of the motor as a function of time.
+        labels = ["I_11", "I_22", "I_33", "I_12", "I_13", "I_23"]
 
-        Parameters
-        ----------
-        lower_limit : float
-            Lower limit of the plot. Default is none, which means that the plot
-            limits will be automatically calculated.
-        upper_limit : float
-            Upper limit of the plot. Default is none, which means that the plot
-            limits will be automatically calculated.
+        fig, ax = plt.subplots(figsize=(12, 8))
 
-        Return
-        ------
-        None
-        """
-        self.motor.I_13.plot(lower=lower_limit, upper=upper_limit)
-
-    def I_23(self, lower_limit=None, upper_limit=None):
-        """Plots I_23 of the motor as a function of time.
+        for label in labels:
+            getattr(self.motor, label).plot(
```

Comment:
I don't think this will going to work as expected.

- The `Function.plot` method does not have a `label` argument. I think you should use the  `Function.plot_1d` instead
- This for loop is simply plotting all the curves one by one in different plots, right? Instead, what we expect from this PR is something very similar to this: https://github.com/RocketPy-Team/RocketPy/pull/547/files#